msal/src/main/java/com/microsoft/identity/client/exception/MsalClientException.java

```diff
@@ -278,7 +278,9 @@ public final class MsalClientException extends MsalException {
      * Configuration error. Native auth app passed with an invalid challenge type.
      */
     public static final String NATIVE_AUTH_INVALID_CHALLENGE_TYPE_ERROR_CODE = "native_auth_invalid_challenge_type";
+    public static final String NATIVE_AUTH_INVALID_CAPABILITY_TYPE_ERROR_CODE = "native_auth_invalid_capability_type";
```

Comment:
We should not use the word "type" for capability. 
```suggestion
    public static final String NATIVE_AUTH_INVALID_CAPABILITY_ERROR_CODE = "native_auth_invalid_capability";
```

Reply:
Done.